code/modules/jobs/job_types/legion.dm

```diff
@@ -704,7 +705,7 @@ commented out pending rework*/
 	suit_store = /obj/item/gun/ballistic/automatic/smg/mp5
 	glasses = /obj/item/clothing/glasses/sunglasses/big
 	backpack_contents = list(
-		/obj/item/ammo_box/magazine/uzim9mm = 2,
+		/obj/item/melee/powered/ripper = 1,
```

Comment:
No, no ripper for explorers on their best loadout

Give them a gladius if you want